src/pyhub/llm/agents/tools/code_interpreter/backends/docker.py

```
@@ -0,0 +1,428 @@
+"""Docker-based code execution backend for Code Interpreter."""
+
+import os
+import json
+import tempfile
+import uuid
+from typing import Any, Dict, Optional
+import docker
+from docker.errors import DockerException, ContainerError, ImageNotFound
```

Ruff (0.11.9)

`docker.errors.DockerException` imported but unused

Remove unused import: `docker.errors.DockerException`

(F401)